src/main/java/jenkins/plugins/logstash/LogstashBuildWrapper.java

```diff
@@ -24,100 +24,63 @@
 
 package jenkins.plugins.logstash;
 
+import java.io.IOException;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+
 import hudson.Extension;
 import hudson.Launcher;
-import hudson.model.BuildListener;
 import hudson.model.AbstractBuild;
 import hudson.model.AbstractProject;
-import hudson.model.BuildableItemWithBuildWrappers;
+import hudson.model.BuildListener;
 import hudson.tasks.BuildWrapper;
 import hudson.tasks.BuildWrapperDescriptor;
 
-import java.io.IOException;
-import java.io.OutputStream;
-import java.util.ArrayList;
-import java.util.List;
-
-import org.kohsuke.stapler.DataBoundConstructor;
-
-import com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsBuildWrapper;
-import com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsBuildWrapper.VarPasswordPair;
-import com.michelin.cio.hudson.plugins.maskpasswords.MaskPasswordsConfig;
-
 /**
- * Build wrapper that decorates the build's logger to insert a
+ * Build wrapper that only indicates to the ConsoleLogFilter
```

Comment:
I might be missing something but this looks like a noop to me -- I don't see any references to ConsoleLogFilter

Reply:
nvm -- found the back-reference in ConsoleLogFilter

Can you reword the docs to state more explicitly that this is just a marker object and ConsoleLogFilter will check for it's presence?

Reply:
done